test/test-spawn.c

```diff
@@ -215,6 +215,88 @@ TEST_IMPL(spawn_fails_check_for_waitpid_cleanup) {
 #endif
 
 
+#ifdef _WIN32
+BOOL is_elevated() {
+  BOOL elevated = FALSE;
+  HANDLE h_token = NULL;
+  if (OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &h_token)) {
+    TOKEN_ELEVATION elevation_token;
```

Comment:
Can you move all of the variable declarations to the top of the function please.